lab_2_keywords_cooccurrence/start.py

```diff
@@ -35,39 +35,33 @@ def read_target_text(file_path: Path) -> str:
     TARGET_TEXT_PATH_ALBATROSS = ASSETS_PATH / 'albatross.txt'
     TARGET_TEXT_PATH_PAIN_DETECTION = ASSETS_PATH / 'pain_detection.txt'
     TARGET_TEXT_PATH_GAGARIN = ASSETS_PATH / 'gagarin.txt'
-
     corpus = {
         'gagarin': read_target_text(TARGET_TEXT_PATH_GAGARIN),
         'albatross': read_target_text(TARGET_TEXT_PATH_ALBATROSS),
         'genome_engineering': read_target_text(TARGET_TEXT_PATH_GENOME),
         'pain_detection': read_target_text(TARGET_TEXT_PATH_PAIN_DETECTION)
     }
 
-    GAGARIN_PROCESSED = process_text(corpus['gagarin'], stop_words)
-    if GAGARIN_PROCESSED:
-        print(get_top_n(GAGARIN_PROCESSED, 10, 5))
-    ALBATROSS_PROCESSED = process_text(corpus['albatross'], stop_words)
-    if ALBATROSS_PROCESSED:
-        print(get_top_n(ALBATROSS_PROCESSED, 10, 5))
-    GENOME_PROCESSED = process_text(corpus['genome_engineering'], stop_words)
-    if GENOME_PROCESSED:
-        print(get_top_n(GENOME_PROCESSED, 10, 5))
-    PAIN_PROCESSED = process_text(corpus['pain_detection'], stop_words)
-    if PAIN_PROCESSED:
-        print(get_top_n(PAIN_PROCESSED, 10, 5))
-
-    STOP_WORDS = load_stop_words(ASSETS_PATH / 'stopwords.json')
+    for text in corpus:
+        processed_text = process_text(text, stop_words)
+        if processed_text:
+            print('text', get_top_n(processed_text, 10, 4), "\n")
 
-    POLISH_PROCESSED = None
-    if STOP_WORDS:
-        POLISH_PROCESSED = process_text(read_target_text(ASSETS_PATH / 'polish.txt'), STOP_WORDS['pl'])
-    if POLISH_PROCESSED:
-        print(get_top_n(POLISH_PROCESSED, 10, 5))
+    polish_text = read_target_text(ASSETS_PATH / 'polish.txt')
+    stop_words = load_stop_words(ASSETS_PATH / 'stopwords.json')  # Incompatible types in assignment
+    # (expression has type "Optional[Mapping[str, Sequence[str]]]", variable has type "List[str]")  [assignment]
+    processed_polish = process_text(polish_text, stop_words)
+    if processed_polish:
+        print('polish_text', get_top_n(process_text(polish_text), 10, 4), "\n")  # Incompatible types in assignment
+        # (expression has type "Optional[Mapping[str, Sequence[str]]]", variable has type "List[str]")  [assignment]
 
-    UNKNOWN_PROCESSED = process_text(read_target_text(ASSETS_PATH / 'unknown.txt'), max_length=8)
-    if UNKNOWN_PROCESSED:
-        print(get_top_n(UNKNOWN_PROCESSED, 10, 5))  # эсперанто
+    unknown_text = read_target_text(ASSETS_PATH / 'unknown.txt')
+    processed_unknown = process_text(unknown_text, stop_words)
+    if processed_unknown:
+        print('unknown_text', get_top_n(process_text(unknown_text), 10, 4), "\n")  # Argument 1 to "get_top_n" has
+        # incompatible type "Optional[Mapping[Tuple[str, ...], float]]";
+        # expected "Mapping[Tuple[str, ...], float]"  [arg-type]
 
-    RESULT = UNKNOWN_PROCESSED
+RESULT = True
```

Comment:
`RESULTS` must be assigned the result of the code execution

Reply:
And the wrong indentation would've gotten away with it, too

Reply:
start.py crashes